tests/unit/pages/Login.spec.js

```diff
@@ -96,21 +96,40 @@ describe("Login.vue", () => {
         const requestOtp = jest
           .spyOn(UserAPIService, "requestOtp")
           .mockImplementation(() => jest.fn());
-
+        const startResendOTPTimer = jest.spyOn(
+          Login.methods,
+          "startResendOTPTimer"
+        );
         mountWrapper();
+
         await setPhoneNumber();
         await requestOTP();
+        await wrapper.setData({ isResendOTPEnabled: false });
 
         // resend OTP
         await wrapper.find('[data-test="resendOTP"]').trigger("click");
 
         await flushPromises();
 
         expect(requestOtp).toHaveBeenCalled();
+        expect(startResendOTPTimer).toHaveBeenCalled();
+        expect(wrapper.vm.resendOTPTimer).toBe(60);
         expect(wrapper.vm.resentOtp).toBe(true);
         expect(wrapper.vm.invalidOtp).toBe(false);
       });
 
+      it("OTP timer", async () => {
+        //To test the Resend OTP Timer
+        //Faketimer is only needed for this test case to advance the time of interval
```

Comment:
you do not need to describe what generic functions are used for - you can delete this

Reply:
It was to test the functioning of the timer.

Reply:
And to check enabling of button